bower.json

```diff
@@ -11,6 +11,7 @@
     "paper-tabs": "PolymerElements/paper-tabs#^2.0.0",
     "paper-styles": "PolymerElements/paper-styles#^2.0.0",
     "underscore": "^1.8.3",
+    "ckeditor": "^4.7.1",
```

Comment:
We don't need this now, I've committed our custom yet "slim" ckeditor to the repo. Still ALOT of files but I'm not sure we have any other option other than publishing our custom ckeditor build to bower itself, might be a good option but it's extra work yada yada.